sdc/datatypes/hpat_pandas_series_functions.py

```diff
@@ -1843,19 +1843,19 @@ def hpat_pandas_series_astype(self, dtype, copy=True, errors='raise'):
 
```

Comment:
dtype two times in caption
:caption: Cast a pandas object to a specified dtype dtype.